addon/components/horizontal-bar-chart.js

```diff
@@ -415,7 +419,7 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
     const maxValueLabelWidth = this._maxWidthOfElements(valueLabelElements);
     const maxGroupLabelWidth = this._maxWidthOfElements(groupLabelElements);
 
-    const maxLabelWidth = this.get('maxLabelWidth');
+    const maxLabelWidth = this.get('maxLabelWidth') || this.get('outerWidth');
```

Comment:
So `getWithDefault` only works when the returned object is exactly `undefined`. So instead of that, this should be an if statement.